tests/backstop_data/Readme.md

```diff
@@ -5,6 +5,9 @@
 
 * The current calendar month has at least one day with some activities
 
+## Contact Section
+* Make sure the civicase is installed with demo data and should have a contact name "Betty Adams" and have some activities for the contact
```

Comment:
Do not expect data to be present with a specific User Name, instead have the expectation by Contact ID.

Reply:
> have some activities
Please mention what is the expected number of activities.